bayesian/api_v1.py

```diff
@@ -152,6 +152,19 @@ def inner(*args, **kwargs):
 #  pass through add_resource_no_matter_slashes
 _resource_paths = []
 
+# START - TOKEN DECODE
+def get_decoded_token():
+    token = request.headers.get('Authorization')
+    if token.startswith('Bearer '):
+        _, token = token.split(' ', 1)
+    try:
+        decoded_token = decode_token(token)
+    except:
```

Comment:
You would probably want to catch `ExpiredSignatureError` here as well to report expired token as stated in https://github.com/fabric8-analytics/fabric8-analytics-server/issues/55